src/ert/gui/main.py

```diff
@@ -143,6 +154,19 @@ def _check_locale():
         return None
 
 
+def _clicked_help_button(menu_label, link):
+    logger = logging.getLogger(__name__)
+    logger.info(f"Pressed help button {menu_label}")
+    functools.partial(webbrowser.open, link)
+
+
+def _clicked_about_button(about_dialog):
+    logger = logging.getLogger(__name__)
+    logger.info("Pressed help button About")
```

Comment:
Could you create a gui test that simply opens and closes the about dialog from the suggester? Thing is, there is no test coverage of that about panel, and now we are putting it up front and center. It would be really embarrassing if it crashes. The user story I am thinking about here could be like this:

```
WHEN I am shown an error in the gui
THEN the suggester gui comes up
AND I can find the version of ert in the about panel.
AND I can copy the error message and go to github to submit an issue by clicking a button.
```

Reply:
Personally, I think adding the above explanation as a comment to such a test could be helpful.

Reply:
Yep. That is reasonable. 👍